Makefile

```diff
@@ -8,3 +8,21 @@ pytest:
 
 pylint:
 	python -m pylint **/*.py
+
+# https://packaging.python.org/tutorials/packaging-projects/
+.PHONY: build
+build:
+	rm dist -r
+	python -m pip install virtualenv
```

Comment:
Why does this use `python` and below targets use `python3`?

Reply:
fixed